.github/scripts/build-linux-aarch64.sh

```diff
@@ -6,7 +6,7 @@ DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
 
 mkdir -p artifacts
 mkdir -p utils
-cp "$(cs get https://github.com/coursier/coursier/releases/download/v2.0.16/cs-aarch64-pc-linux)" utils/cs
+cp "$(cs get https://github.com/VirtusLab/coursier-m1/releases/download/v2.1.4/cs-aarch64-pc-linux)" utils/cs
```

Comment:
I changed the url here
* from coursier/coursier (https://github.com/coursier/coursier/releases/)
* to VirtusLab/coursier-m1 (https://github.com/VirtusLab/coursier-m1/releases)

is that correct?

Reply:
@mkurz Thank you for updating it. Yes, that is correct, as Coursier for M1 is built in the `coursier-m1` repository. When the CI passes, I will merge this PR.